src/Data/Rational/Unnormalised/Properties.agda

```diff
@@ -792,8 +792,8 @@ neg⇒nonZero (mkℚᵘ (-[1+ _ ]) _) = _
 +-inverse : Inverse _≃_ 0ℚᵘ -_ _+_
 +-inverse = +-inverseˡ , +-inverseʳ
 
-+-cancelˡ : ∀ {r p q} → r + p ≃ r + q → p ≃ q
-+-cancelˡ {r} {p} {q} r+p≃r+q = begin-equality
++-cancelˡ : LeftCancellative _≃_ _+_
++-cancelˡ r p q r+p≃r+q = begin-equality
```

Comment:
How should this type change be reflected in the `CHANGELOG`, given that this is now the 'original' intention?

Reply:
I think what's in the `CHANGELOG` now reflects this change quite well.